web-app/src/pages/Login/LoginPage.tsx

```diff
@@ -112,7 +112,8 @@ export default function AuthComponent() {
 			setAuthData((prevState) => {
 				return { ...prevState, isError: false, helperText: "Login successful!" };
 			});
-			navigate("/pipelines");
+			window.location.href = "/pipelines";
```

Comment:
Note to future self: This is to prevent the css from being destroyed when navigating from the login page to the main app.